name-service/js/package.json

```diff
@@ -21,12 +21,12 @@
     "dev": "tsc && node --trace-warnings dist/transfer.js",
     "build": "tsc",
     "prepublish": "tsc",
-    "lint": "yarn pretty && eslint --max-warnings 0 'src/*.ts'",
-    "lint:fix": "yarn pretty:fix && eslint 'src/*.ts' --fix",
+    "lint": "npm run pretty && eslint --max-warnings 0 'src/*.ts'",
+    "lint:fix": "npm run pretty:fix && eslint 'src/*.ts' --fix",
     "pretty": "prettier --check '{src/*.ts,test/*/*.ts}'",
     "pretty:fix": "prettier --write '{src/*.ts,test/*/*.ts}'",
-    "doc": "yarn typedoc src/index.ts",
-    "test": "yarn test:unit && yarn test:e2e",
+    "doc": "npm run typedoc src/index.ts",
+    "test": "npm run test:unit && npm run test:e2e",
```

Comment:
why the change to npm instead of either pnpm or leaving it as-is?

Reply:
pnpm fails if yarn is mentioned in the scripts, but works with npm. Since the other packages use npm when calling other scripts, I decided for consistency. We could certainly change it to pnpm though!